New Version with corrected links

JustFxDev opened the pull request:
After take over from David a did not insert the new download links. This has been fixed.

JustFxDev committed:
New Version with corrected links

After take over from David a did not insert the new download links. This has been fixed.

aljawaid commented on plugins.json:
```diff
@@ -476,21 +476,21 @@
-        "description": "Allow boards to be sorted by card due date.",
-        "download": "https://raw.githubusercontent.com/JustFxDev/kanboard-duedate/master/releases/DueDate.zip",
+        "description": "Switch sorting boards by card due date, last modification of tasks oder default view",
+        "download": "https://github.com/JustFxDev/kanboard-duedate/archive/refs/tags/V1.2.0.zip",
```
Your archive folder is incorrect

aljawaid reviewed (changes requested):
Your archive folder is incorrect, users will struggle to install/update this plugin

JustFxDev committed:
One more change

I am going insane. The download archive was still not functional.

But now it should finally work. Sorry. My first contribution to Kanboard / PlugIns.
